assets/js/add-another.js

```diff
@@ -58,9 +59,11 @@ AddAnother.prototype.updateAttributes = function(index, item) {
 }
 
 AddAnother.prototype.createRemoveButton = function(item) {
-  item.append(
-    '<button type="button" class="govuk-button govuk-button--secondary add-another__remove-button">Remove</button>'
-  )
+  item
+    .find(this.removeContainerSelector)
+    .append(
+      '<button type="button" class="govuk-button govuk-button--secondary add-another__remove-button govuk-!-margin-left-3">Remove</button>'
```

Comment:
Not a massive fan of sticking HTML in strings like this as they tend to get missed by lint like tooling etc. Maybe, tackle it if we end up doing more of it?

Reply:
yes, we've inherited this from the hmcts add another.. will consider this if we need to do more of it.